src/main/java/com/brainpix/post/controller/IdeaMarketController.java

```diff
@@ -0,0 +1,61 @@
+package com.brainpix.post.controller;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.brainpix.api.ApiResponse;
+import com.brainpix.post.dto.GetIdeaCommentListDto;
+import com.brainpix.post.dto.GetIdeaDetailDto;
+import com.brainpix.post.dto.GetIdeaListDto;
+import com.brainpix.post.dto.GetPopularIdeaListDto;
+import com.brainpix.post.service.IdeaMarketService;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@RequestMapping("/api/ideas")
+@RequiredArgsConstructor
+@Slf4j
+public class IdeaMarketController {
+	private final IdeaMarketService ideaMarketService;
+
+	@GetMapping
+	public ResponseEntity<ApiResponse<GetIdeaListDto.Response>> getIdeaList(
+		GetIdeaListDto.Request request,
+		Pageable pageable
+	) {
+		GetIdeaListDto.Response response = ideaMarketService.getIdeaList(request, pageable);
```

Comment:
저는 controller -> service 와 같이 레이어끼리도 dto로 묶어서 관리하는 편이에요
그렇게 하면 추후 리팩토링시 Dto와 converter 레이어만 관리해주면 됩니다 참고해주시면 감사하겠습니다!
```suggestion
        GetIdeaListDto.Parameter param = GetIdeaListConverter.convertToParameter(request, paeable);
		GetIdeaListDto.Response response = ideaMarketService.getIdeaList(param);
```